### DISCUSSION TITLE
cSpot?

### DISCUSSION THREAD

**Discussion by nagyrobi** [sle118/squeezelite-esp32#235] [Q&A] [answered]
I just found in the firmware the cSpot feature (squeezelite-esp32-master-v4.3-I2S-4MFlash-16-21271). Don't see any info in the readme...
How's it supposed to work?

I have an ESP32-A1S board (rev b248) which works fine with LMS, but when selecting it from Spotify app, it looks like it's taking the playback over, but no sound comes out. Am I doing something wrong? I tried pausing LMS playback first, powercycling the board, exchanged binary from 16-bit to 32-bit etc. but no change.

**Comment by AxelJust** [comment, marked as answer]
My experience with cSpot is that it sometimes requires to turn up the voliume on the playback devise / the app.
Have you tried that?

**Reply by nagyrobi** [reply]
Ah good tip. Indeed the volume has to be turned up and indeed it starts playing for a couple of seconds but then it stops.  The service gets also confused on the mobile client too, as for a while it still shows as playing but then the cursor stops there too.

**Reply by nagyrobi** [reply]
Tested the wrong version. [All sorted now.](https://github.com/sle118/squeezelite-esp32/issues/237)
